tools/noise/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+project(noise-tools C)
+
+find_library(M_LIBRARY m)
+
+add_executable(dt-noiseprofile noiseprofile.c)
+target_link_libraries(dt-noiseprofile ${M_LIBRARY})
+
+install(TARGETS
+            dt-noiseprofile
+        DESTINATION
+            ${CMAKE_INSTALL_LIBEXECDIR}/darktable/tools
+        COMPONENT
+            DTApplication)
+
+if (UNIX AND NOT WIN32)
+    install(PROGRAMS
+                dt-gen-noiseprofile
+            DESTINATION
+                ${CMAKE_INSTALL_LIBEXECDIR}/darktable/tools
+            COMPONENT
+                DTApplication)
+
+    install(FILES
+                subr.sh
+                profiling-shot.xmp
+            DESTINATION
+               ${CMAKE_INSTALL_LIBEXECDIR}/darktable/tools
+            COMPONENT
+                DTApplication)
+
+    macro(install_symlink _filepath _sympath)
+        install(CODE "
+            if (\"\$ENV{DESTDIR}\" STREQUAL \"\")
+                execute_process(COMMAND \"${CMAKE_COMMAND}\" -E create_symlink
+                                ${_filepath}
+                                ${_sympath})
+            else ()
+                execute_process(COMMAND \"${CMAKE_COMMAND}\" -E create_symlink
+                                ${_filepath}
+                                \$ENV{DESTDIR}/${_sympath})
+            endif ()
+        ")
+        install(CODE "message(\"-- Creating symlink: ${_sympath} -> ${_filepath}\")")
+    endmacro(install_symlink)
+
+    install_symlink("${CMAKE_INSTALL_FULL_LIBEXECDIR}/darktable/tools/dt-gen-noiseprofile" "${CMAKE_INSTALL_FULL_BINDIR}/dt-gen-noiseprofile")
```

Comment:
What is that good for? To have the main script in people's $PATH?

Reply:
I can remove that if you don't want that.